README.md

```diff
@@ -6,6 +6,7 @@ This repository contains the inventory service for an e-commerce business. The s
 [![Build Status](https://travis-ci.com/NYU-DevOps-Inventory-Team/inventories.svg?branch=main)](https://travis-ci.com/NYU-DevOps-Inventory-Team/inventories)
 [![codecov](https://codecov.io/gh/NYU-DevOps-Inventory-Team/inventories/branch/main/graph/badge.svg?token=5NLUQE5NIR)](https://codecov.io/gh/NYU-DevOps-Inventory-Team/inventories)
 
+
```

Comment:
obviously one blank line doesn't matter at all, but this is a good learning moment to see why you should always do a git status or look through your changelist on the lefthand panel of the version control tab on VSCode to make sure you are only committing files you need to.